modules/tensor_mechanics/test/tests/combined_creep_plasticity/creepWithPlasticity.i

```diff
@@ -0,0 +1,182 @@
+#
+# This test is Example 2 from "A Consistent Formulation for the Integration
+#   of Combined Plasticity and Creep" by P. Duxbury, et al., Int J Numerical
+#   Methods in Engineering, Vol. 37, pp. 1277-1295, 1994.
+#
+# The problem is a one-dimensional bar which is loaded from yield to a value of twice
+#   the initial yield stress and then unloaded to return to the original stress. The
+#   bar must harden to the required yield stress during the load ramp, with no
+#   further yielding during unloading. The initial yield stress (sigma_0) is prescribed
+#   as 20 with a plastic strain hardening of 100. The mesh is a 1x1x1 cube with symmetry
+#   boundary conditions on three planes to provide a uniaxial stress field.
+#
+#  In the PowerLawCreep model, the creep strain rate is defined by:
+#
+#   edot = A(sigma)**n * exp(-Q/(RT)) * t**m
+#
+#   The creep law specified in the paper, however, defines the creep strain rate as:
+#
+#   edot = Ao * mo * (sigma)**n * t**(mo-1)
+#      with the creep parameters given by
+#         Ao = 1e-7
+#         mo = 0.5
+#         n  = 5
+#
+#   thus, input parameters for the test were specified as:
+#         A = Ao * mo = 1e-7 * 0.5 = 0.5e-7
+#         m = mo-1 = -0.5
+#         n = 5
+#         Q = 0
+#
+#   The variation of load P with time is:
+#       P = 20 + 20t      0 < t < 1
+#       P = 40 - 40(t-1)  1 < t 1.5
+#
+#  The analytic solution for total strain during the loading period 0 < t < 1 is:
+#
+#    e_tot = (sigma_0 + 20*t)/E + 0.2*t + A * t**0.5  * sigma_0**n * [ 1 + (5/3)*t +
+#               + 2*t**2 + (10/7)*t**3 + (5/9)**t**4 + (1/11)*t**5 ]
+#
+#    and during the unloading period 1 < t < 1.5:
+#
+#    e_tot = (sigma_1 - 40*(t-1))/E + 0.2 + (4672/693) * A * sigma_0**n +
+#               A * sigma_0**n * [ t**0.5 * ( 32 - (80/3)*t + 16*t**2 - (40/7)*t**3
+#                                  + (10/9)*t**4 - (1/11)*t**5 ) - (11531/693) ]
+#
+#         where sigma_1 is the stress at time t = 1.
+#
+#  Assuming a Young's modulus (E) of 1000 and using the parameters defined above:
+#
+#    e_tot(1) = 2.39734
+#    e_tot(1.5) = 3.16813
+#
+#
+#   The numerically computed solution is:
+#
+#    e_tot(1) = 2.39718         (~0.006% error)
+#    e_tot(1.5) = 3.15555       (~0.40% error)
+#
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 1
+  ny = 1
+  nz = 1
+[]
+
+[Modules/TensorMechanics/Master]
+  [all]
+    strain = FINITE
+    incremental = true
+    add_variables = true
+    generate_output = 'stress_yy elastic_strain_yy creep_strain_yy plastic_strain_yy'
+  []
+[]
+
+[Functions]
+  [top_pull]
+    type = PiecewiseLinear
+    x = '  0   1   1.5'
```

Comment:
Is the closing quote on line 83 some nonstandard character? It's showing up with a different background color for me on my github. This could be nothing, but please check.

Reply:
Just a regular old quote mark.